components/ProductCard.vue

```diff
@@ -72,9 +72,11 @@ export default Vue.extend({
       if (!this.$store) {
         return 0
       }
+
       const cart = this.$store.state.cart
```

Comment:
Should be a getter

Reply:
Ah, hm, are `this.$store.state` never to be used in your opinion or are there cases?

I'm a bit confused because getters are like computed and should be used when values should be modified, or am I wrong?

So I guess I should use a getter here, and include the `find` filter in the getter. 

But then these getter from `store/products.ts` should not be used, and `this.$store.state.products` should be used instead?

```
    isLoading(state: ProductsState): boolean {
      return state.loading
    },
    didShowMore(state: ProductsState) {
      return state.didShowMore
    },
    getProducts(state: ProductsState): Array<Product> {
      return state.recieved
    },
```

Reply:
Getters are like computed, yes! But it's also good for creating an API endpoint of sorts. Maybe we don't want to keep the cart in `this.$store.state.cart` in the future, then we can just change what data the getter presents 👍 

> So I guess I should use a getter here, and include the find filter in the getter.

That sounds like a good idea, but I'm not sure how we can get `this.card.code` into the getter

> But then these getter from store/products.ts should not be used, and this.$store.state.products should be used instead?

Not sure what you mean, but I think we can keep them

Reply:
I understand what you mean, but right now I see no problem using `this.$store.state.cart` if we want to get the data.

It can be changed later with search/replace, but for now I think it's quite nice to know when the data is raw and not modified.